docs/UserGuide.md

```diff
@@ -93,11 +93,12 @@ The `Code Panel` allows users to see the code attributed to a specific author. C
 * Non-trivial code segments that are not written by the selected author are hidden by default, but you can toggle them by clicking on the `...` icon.
 
 ### Commits Panel
-![Commits panel](/docs/images/commits-panel.png)
+<img src="/docs/images/commits-panel.png" alt="commits panel" width="468" height="424">
 
-The `Commits Panel` allows users to see the commits attributed to a specific author. Hold `Command`&#8984; (MacOS) or `Ctrl` (Others) and click on the ramp chart in the `Chart Panel` to select the time range where you want to display the `Commit Panel` for on the right. <br>
+The `Commits Panel` allows users to see the commits attributed to a specific author. Hold `Command`&#8984; **(MacOS)** or `Ctrl` **(other OSes)** and click on the ramp chart in the `Chart Panel` to select the time range where you want to display the `Commit Panel` for on the right. <br>
 
 ![Opening commits panel](/docs/images/opening-commits-panel.gif)
+<img src="/docs/images/commits-panel.png" alt="commits panel" width="468" height="424">
```

Comment:
why is the same image being shown twice? Also, can set just either width / height so that it looks more proportionate.

Reply:
mb, forgot to remove that line

Reply:
@eugene is the size for those commits panel image and the .gif ok?

Reply:
yes, looks good to me 👍